src/components/Map/LeafletDrawTools.js

```diff
@@ -85,6 +81,50 @@ export default function LeafletDrawTools(props) {
       'stroke-width': 5
     }
   };
+  const maxPolygonAreaSize = 100000000; // 1000 sq km
+  const [areaNumber, setAreaNumber] = useState(1);
+  const areaNumberRef = useRef();
+  areaNumberRef.current = areaNumber;
```

Comment:
It doesn't look like this is being used now (looks like you're setting state instead)